walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/index.html

```diff
@@ -99,10 +99,9 @@
       <input type="text" name="" placeholder="First Name">
       <input type="text" name="" placeholder="Last Name">
       <input type="text" name="" placeholder="Email">
-      <button type="button" name="button" onclick="message()">Send</button>
+      <button class="form-button" type="button" name="button">Send</button>
```

Comment:
Please do the same for the other 6 `onclick="animateScrollTop..."`

Reply:
Done, did for every a link